app/views/DR/UserInfoScreen/index.js

```diff
@@ -119,7 +119,7 @@ export default function UserInfo({ navigation }) {
           closeDialog(false);
           if (positive) {
             if (usage === 'mySelf') {
-              storeData('positive', positive);
+              storeData('COVID_POSITIVE', positive);
```

Comment:
As you changed this, you have to change the 'positive' key storage wherever it appears. It appears in settings i guess.